views/js/qtiXmlRenderer/renderers/ResponseProcessing.js

```
@@ -39,10 +39,12 @@ define([
                         }
                     }
 
+                    const _score = 'SCORE';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)